app/views/thesis/new.html.erb

```diff
@@ -124,6 +124,7 @@
     </div>
 
     <%= f.association :copyright, as: :select,
+                                  selected: 2,
```

Comment:
I wonder if for the edit view we can do something like `selected: 2 if license.nil?` (not real code). My reasoning is that because of the registrar data load pre-creating theses, I think most users hit the edit form and not the new form.

Reply:
Ah, good point. I'll see what options we have there.